release.Dockerfile

```diff
@@ -1,33 +1,42 @@
+FROM python:3.9-slim AS builder
+
+# Copying argilla distribution files
+COPY dist/*.whl /packages/
+RUN python -m venv /my-virtual-env
+ENV PATH="/my-virtual-env/bin:$PATH"
+RUN apt-get update && \
+    apt-get install -y python-dev-is-python3 libpq-dev gcc && \
+    for wheel in /packages/*.whl; do pip install "$wheel"[server,postgresql]; done && \
+    apt-get remove -y python-dev-is-python3 libpq-dev gcc && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm -rf /packages
+
+
 FROM python:3.9-slim
 
 # Environment Variables
-ENV ARGILLA_HOME_PATH=/var/lib/argilla
+ENV ARGILLA_HOME_PATH=/home/worker/argilla
```

Comment:
`/var/lib` doesn't allow non-root ownership afaik

Reply:
:+1 

May this change affect old containers updating to this image?